advanced/am-pattern-3/templates/am/km/wso2am-pattern-2-am-km-deployment.yaml

```diff
@@ -99,6 +99,9 @@ spec:
         - name: wso2am-pattern-3-am-km-conf
```

Comment:
@Chathuru why do we need to override the entry point here?

Reply:
Fix https://github.com/wso2/kubernetes-apim/pull/365/commits/3588e9f5044c3947741ef019ecda0ee150daad18